modules/menuItems.js

```diff
@@ -432,6 +432,32 @@ var menuTempl = function(webviews) {
         ]
     })
 
+    let languageMenu =
+    Object.keys(i18n.options.resources).map(lang_code => {
+        menuItem = {
+            label: lang_code,
+            click: function(){
+                let mainWindow = Windows.getByType('main');
+                mainWindow.webContents.executeJavaScript(
+                    `TAPi18n.setLanguage("${lang_code}");`
```

Comment:
Thanks for the hint, @frozeman. However I tried calling it in `menuItem.js` and calling it in webpage, both failed. First attempt:
```js
        click: function(){
                let mainWindow = Windows.getByType('main');
                mainWindow.webContents.executeJavaScript(
                    `TAPi18n.setLanguage('${lang_code}');`
                );
                ipc.send('backendAction_setLanguage', lang_code);
            }
```
which gives 
```
TypeError: ipc.send is not a function
    at menuItem.click (/home/chihchengliang/ether/mist/modules/menuItems.js:444:21)
    at MenuItem.click (/home/chihchengliang/npm-global/lib/node_modules/electron-prebuilt/dist/resources/electron.asar/browser/api/menu-item.js:81:16)
    at Function.delegate.executeCommand (/home/chihchengliang/npm-global/lib/node_modules/electron-prebuilt/dist/resources/electron.asar/browser/api/menu.js:119:40)
```
in electron console and crashed.  Checked the doc, `ipcMain` has no function `send`. It seems I can't send from main process.
I see ipc.send can be called from popWindow, so tried sending from the webpage too.
```js
                mainWindow.webContents.executeJavaScript(
                    `TAPi18n.setLanguage('${lang_code}');
                    ipc.send('backendAction_setLanguage', '${lang_code}');`
                );
```
which throws `Uncaught ReferenceError: ipc is not defined` in browser console: